### DISCUSSION TITLE
No colors in matplotlib with Frontend

### DISCUSSION THREAD

**Discussion by gutierrad** [mozman/ezdxf#849] [Q&A] [answered]
Hi all,

First of all, let me thank @mozman for the work. It is awesome.
I come here today because after updating from version 0.16.5 (I know, too old) to version 1.0.2, the objects in a dxf that I have generated with ezdxf have the right color in AutoCAD, but in the image generated using matplotlib they are white.
I am using ByLayer colors, and I set the colors from the Colors enum.
The problem occurs with all types of objects.
Has this occurred to anybody?

Thank you very much,
David

**Comment by mozman** [comment, marked as answer]
Please provide a (zipped) example DXF.

**Reply by gutierrad** [reply]
Dear Manfred,

thank you very much for your prompt response.
After publishing my question, I have continued trying to debug the problem, but your request for the DXF gave me the clue to find the problem: I was opening in CAD a different file.
I am creating two different (but very similar) files at the same time.
It turns out I was mistakenly not creating the layers in the second one because of the very similar variable name.

I am a bit new to DXF debugging and I am still trying to understand all the steps.

Again, thank you very much for your help.

Best,
David
